lib/src/widgets/calendar_header.dart

```diff
@@ -56,7 +58,7 @@ class CalendarHeader extends StatelessWidget {
               padding: headerStyle.leftChevronPadding,
             ),
           Expanded(
-            child: GestureDetector(
+            child: headerTitleBuilder != null ? headerTitleBuilder : GestureDetector(
```

Comment:
Let's change this line to `headerTitleBuilder?.call(context, focusedMonth) ??`, as the line above results in runtime error (missing params).

Reply:
Understood

Reply:
done!